src/main/java/it/pagopa/pn/delivery/svc/search/NotificationRetrieverService.java

```diff
@@ -220,6 +223,77 @@ public InternalNotification getNotificationInformation(String iun, boolean withT
 		}
 	}
 
+	private void computeIuvToReturn(InternalNotification notification) {
+		log.debug( "Set IUV to return" );
```

Comment:
Log errato? E' compute IUV to return.
Utilizzerei compute NoticeCode invece di IUV.